provider/k8s.go

```diff
@@ -390,11 +385,25 @@ func NewLocalExecutor(config Config, logger *zap.SugaredLogger) (Executor, error
 	}, nil
 }
 
-func (kube KubernetesExecutor) isDefaultImage() bool {
+type KubernetesExecutor struct {
+	logger *zap.SugaredLogger
+	image  string
+}
+
+func (kube *KubernetesExecutor) isDefaultImage() bool {
 	return strings.Contains(kube.image, cfg.PandasBaseImage)
```

Comment:
This is technically not correct right? You could have a base image that contains the same name as the default image. I'd recommend just leaving a comment about that and leaving it as is.

Reply:
Added